src/main/resources/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStep/help-script.html

```diff
@@ -9,4 +9,33 @@
     Otherwise the system default shell will be run, using the <code>-xe</code> flags
     (you can specify <code>set +e</code> and/or <code>set +x</code> to disable those).
     </p>
+    <p>A few examples of usage include:</p>
+
+    <b>Creating an output folder</b>
+    <p><code>sh "mkdir -p output"</code></p>
+
+    <b>Returning the current directory Pipeline is running in.</b>
+    <p><code>sh "ls -la ${pwd()}"</code></p>
+
+    <b>Making HTTP requests </b>
+    <p><code>sh "curl -X POST --data-urlencode \'payload=${payload}\' ${slackURL}"</code></p>
+    
+    <b>Running tests in the same workspace that the project was built </b>
+    <p><code>sh 'mvn test'</code></p>
+
```

Comment:
I would be great to have a declarative multiline example (3 single quote to begin and ends), maybe with this one, something like :

```suggestion
   <b>Using multi-line syntax to run several lines of script:</b>
    <p>
        <pre>
    sh '''
    echo "This is a first line of script"
    # This is a script comment
    echo "This is a second line"
    '''
        </pre>
    </p>
```

Reply:
Right, this looks good. I think i'll just go ahead and commit this suggestion, if that's okay.